packages/apps/tasks/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "tasks",
+  "version": "0.1.0",
+  "description": "tasks - a DXOS application",
+  "private": true,
+  "scripts": {
+    "build": "tsc --noEmit && vite build",
+    "deploy": "dx app publish",
+    "preview": "vite preview",
+    "serve": "vite",
+    "storybook": "start-storybook -p 9009 --no-open",
+    "prebuild": "dxtype src/proto/schema.proto src/proto/gen/schema.ts"
```

Comment:
I think I need to be running `dxtype` on `serve` as well in order to pick up protobuf changes during development. What's the appropriate incantation? Couldn't find any examples that looked right to me.

Reply:
The `project.json` is the key file for packages within the monorepo, you'll notice a lot of the other apps are limited on scripts and prefer targets in the nx config instead because it allows config to be colocated. The nx config files allow you to define dependencies between targets as well, so with that it should be easy to say "run prebuild before serve" (if that's not already a monorepo-wide default). Happy to walk you through any of the Nx stuff you have questions about and also would love if you had a moment to update the repo guide with any key things you think are missing wrt Nx info.